std/algorithm/searching.d

```diff
@@ -3160,36 +3187,38 @@ if (isForwardRange!R1 && isForwardRange!R2)
 }
 
 /// Use $(REF only, std,range) to find single elements:
-@safe pure nothrow unittest
+@safe pure nothrow @nogc unittest
 {
     import std.range : only;
-    assert([1, 2, 3, 4].findSplitBefore(only(3))[0] == [1, 2]);
+    import std.array : staticArray;
+    assert([1, 2, 3, 4].staticArray[].findSplitBefore(only(3))[0] == [1, 2].staticArray);
```

Comment:
As elsewhere, it seems to me that this change makes the unit test worse as documentation.